docs/topics/js/javascript-dce.md

```diff
@@ -15,6 +21,26 @@ Unused declarations can appear in cases like:
 The Kotlin Multiplatform Gradle plugin handles DCE automatically when you build a **production bundle**, for example by using the
 `browserProductionWebpack` task. **Development bundling** tasks (like `browserDevelopmentWebpack`) don't include DCE.
 
+## DCE and JavaScript IR compiler
+
+The new way of applying dead code elimination that works with the IR compiler is:
+
+* DCE is disabled when compiling for development, which corresponds to the following Gradle tasks:
+  * `browserDevelopmentRun`
+  * `browserDevelopmentWebpack`
+  * `nodeDevelopmentRun`
+  * `compileDevelopmentExecutableKotlinJs`
+  * `compileDevelopmentLibraryKotlinJs`
+  * Other Gradle tasks including "development" in their name
+* DCE is enabled when compiling for production, which corresponds to the following Gradle tasks:
+  * `browserProductionRun`
+  * `browserProductionWebpack`
+  * `compileProductionExecutableKotlinJs`
+  * `compileProductionLibraryKotlinJs`
+  * Other Gradle tasks including "production" in their name
+
+With the @JsExport annotation, you can specify the declarations you want DCE to treat as roots.
```

Comment:
No, we call "roots" the declarations that we want to treat as entry points during dead code elimination, so that they and whatever declarations they transitively reference are not eliminated.

Reply:
But yeah, we may prefer some term that's easier to understand for ordinary users than some compiler lingo.

Reply:
I'll leave it as "roots" being the current used concept.